src/aat/java/uk/gov/hmcts/ccd/datastore/befta/DataStoreTestAutomationAdapter.java

```diff
@@ -0,0 +1,127 @@
+package uk.gov.hmcts.ccd.datastore.befta;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import uk.gov.hmcts.befta.BeftaMain;
+import uk.gov.hmcts.befta.DefaultTestAutomationAdapter;
+import uk.gov.hmcts.befta.data.UserData;
+import uk.gov.hmcts.befta.exception.FunctionalTestException;
+
+public class DataStoreTestAutomationAdapter extends DefaultTestAutomationAdapter {
+
+    private Logger logger = LoggerFactory.getLogger(DataStoreTestAutomationAdapter.class);
+
+    private static final String[] TEST_DEFINITIONS_NEEDED_FOR_TA = {
+            "src/aat/resources/CCD_CNP_27.xlsx",
+            "src/aat/resources/CCD_CNP_27_AUTOTEST1.xlsx",
+            "src/aat/resources/CCD_CNP_27_AUTOTEST2.xlsx",
+            "src/aat/resources/CCD_CNP_RDM5118.xlsx",
+
+            "src/aat/resources/CCD_BEFTA_JURISDICTION1.xlsx",
+            "src/aat/resources/CCD_BEFTA_JURISDICTION2.xlsx",
+            "src/aat/resources/CCD_BEFTA_JURISDICTION3.xlsx"
+    };
+
+    private static final String[][] CCD_ROLES_NEEDED_FOR_TA = {
+            { "caseworker-autotest1", "PUBLIC" },
```

Comment:
same as above, config rather than hardcoding?

Reply:
having it in config improves the visibility of it. This is an important piece of configuration that is currently "hidden" in the code